hopscotch/README.md

```diff
@@ -0,0 +1,21 @@
+# Tour
+A demonstration of https://testspace.com using the **Hopscotch** framework from LinkedIn.  
+
+To learn more about Hopscotch and the API, check out the repo: https://github.com/linkedin/hopscotch.Note that Hopscotch is **no longer** supported. 
+
+For usage information, go [here](https://s2technologies.github.io/hopscotch/).
+
+## Files
+The hopscotch files: 
+  - `hopscotch.min.css`
+  - `hopscotch.min.js`
```

Comment:
Since we are hosting `hopscotch` (at s2) have you considered serving their files directly from there?
```
https://s2technologies.github.io/hopscotch/js/hopscotch-0.2.6.min.js
https://s2technologies.github.io/hopscotch/css/hopscotch-0.2.6.min.css
```